lisa/wlgen/rta.py

```diff
@@ -180,11 +180,18 @@ def by_profile(cls, target, name, profile, res_dir=None, default_policy=None,
 
         calibration = self._process_calibration(calibration)
 
+        if target.is_rooted:
+            lock_pages = True
+        else:
+            lock_pages = False
+            logger.debug('rtapp global option "lock_pages" disabled when non-root')
```

Comment:
Since this could affect results, would not be better to warn here?

Reply:
I'm not sure if warning just because somebody runs as non-root is a good thing to do. So far rt-app was not using mlockall for any of the CFS stuff and we did not have lots of issues. That said, if it really makes a noticeable difference on some setups, it would make sense to actually warn.